src/Makefile.am

```diff
@@ -77,6 +74,15 @@ libfapolicyd_la_SOURCES += \
 
 endif
 
+if WITH_DEB
+libfapolicyd_la_SOURCES += library/deb-backend.c
+fapolicyd_CFLAGS += -DLIBDPKG_VOLATILE_API
```

Comment:
Something I am worried about is the dpkg API seems pretty unstable. I suspect there will be some issues with different versions of `dpkg-dev` on different distros. They force this acknowledgement of its instability here.

Reply:
I guess we'll find out as we go along.